Gems/MiniAudio/Code/Source/Clients/MiniAudioPlaybackComponentConfig.cpp

```diff
@@ -24,9 +24,27 @@ namespace MiniAudio
                 ->Field("Auto-follow", &MiniAudioPlaybackComponentConfig::m_autoFollowEntity)
                 ->Field("Loop", &MiniAudioPlaybackComponentConfig::m_loop)
                 ->Field("Spatialization", &MiniAudioPlaybackComponentConfig::m_enableSpatialization)
+                ->Field("Fixed Direction", &MiniAudioPlaybackComponentConfig::m_fixedDirection)
+                ->Field("Direction", &MiniAudioPlaybackComponentConfig::m_direction)
                 ->Field("Attenuation Model", &MiniAudioPlaybackComponentConfig::m_attenuationModel)
+                ->Field("Directional Attenuation Factor", &MiniAudioPlaybackComponentConfig::m_directionalAttenuationFactor)
                 ->Field("Min Distance", &MiniAudioPlaybackComponentConfig::m_minimumDistance)
                 ->Field("Max Distance", &MiniAudioPlaybackComponentConfig::m_maximumDistance)
+                ->Field("Inner Cone Angle (Degrees)", &MiniAudioPlaybackComponentConfig::m_innerAngleInDegrees)
+                    ->Attribute(AZ::Edit::Attributes::Min, 0.0f)
+                    ->Attribute(AZ::Edit::Attributes::Step, 1.0f)
+                    ->Attribute(AZ::Edit::Attributes::Max, 360.0f)
+                    ->Attribute(AZ::Edit::Attributes::Suffix, " degrees")
+                ->Field("Outer Cone Angle (Degrees)", &MiniAudioPlaybackComponentConfig::m_outerAngleInDegrees)
+                    ->Attribute(AZ::Edit::Attributes::Min, 0.0f)
+                    ->Attribute(AZ::Edit::Attributes::Step, 1.0f)
+                    ->Attribute(AZ::Edit::Attributes::Max, 360.0f)
+                    ->Attribute(AZ::Edit::Attributes::Suffix, " degrees")
+                ->Field("Outer Volume (%)", &MiniAudioPlaybackComponentConfig::m_outerVolume)
+                    ->Attribute(AZ::Edit::Attributes::Min, 0.0f)
+                    ->Attribute(AZ::Edit::Attributes::Step, 1.0f)
+                    ->Attribute(AZ::Edit::Attributes::Max, 100.0f)
+                    ->Attribute(AZ::Edit::Attributes::Suffix, " %")
```

Comment:
ditto here, please remove all Edit attributes from the SerializeContext declaration of the class.  These belong in the `Reflect()` method where the  EditContext version of the class is declared - usually in the EditorMiniAudioPlaybackComponentConfig.cpp file if it exists